composer.json

```diff
@@ -18,22 +18,21 @@
     "require": {
         "composer/installers": "^2.0",
         "drupal/admin_toolbar": "^3",
-        "drupal/config_ignore": "2.x-dev",
-        "drupal/config_split": "^1",
-        "drupal/core-composer-scaffold": "^10",
-        "drupal/core-recommended": "^10",
+        "drupal/config_ignore": "^3",
+        "drupal/config_split": "^2",
+        "drupal/core-composer-scaffold": "^11",
+        "drupal/core-recommended": "^11",
         "drupal/devel": "^5",
-        "drupal/workbench": "^1",
-        "drupal/workbench_tabs": "^1",
-        "drush/drush": ">=11"
+        "drupal/workbench": "^1.5",
+        "drupal/workbench_tabs": "^1.8",
+        "drush/drush": "^13"
     },
     "require-dev": {
         "behat/behat": "^3.12",
-        "behat/mink-goutte-driver": "^2.0",
         "dmore/behat-chrome-extension": "^1.4",
-        "drupal/core-dev": "^10",
-        "drupal/drupal-extension": "^5@alpha",
-        "palantirnet/the-build": "^4@beta"
+        "drupal/core-dev": "^11",
+        "drupal/drupal-extension": "^5",
+        "palantirnet/the-build": "dev-drupal11"
```

Comment:
TODO: Change this back to a stable version constraint after the `drupal11` branch is merged into the-build, and there is a new release.